docs/consent/README.md

```diff
@@ -0,0 +1,38 @@
+<h1 align="center"><img src="https://make.hmn.md/altis/Altis-logo.svg" width="89" alt="Altis" /> Consent</h1>
+
+<p align="center">Enhanced user privacy and consent controls for <strong><a href="https://altis-dxp.com/">Altis</a></strong>.</p>
+
+<p align="center"><a href="https://packagist.org/packages/altis/consent"><img alt="Packagist Version" src="https://img.shields.io/packagist/v/altis/consent.svg"></a></p>
```

Comment:
This part should be removed in favour of a regular title.
```suggestion
# Consent API
```

Reply:
Changing to `Altis Consent` because we already have a `Consent API` heading/page.